templates/year/Makefile.conf.sample

```diff
@@ -5,11 +5,11 @@ SOAPCLIENT := fks-soapclient.sh
 STATS_SH := fks-dakos-stats.sh
 
 # path to XSL stylesheets
-XSL_SIGNATURES := /usr/local/share/fks/texsignatures.xsl
-#XSL_RESULTS := /usr/local/share/fks/texresults-fykos.xsl
-#XSL_RESULTS := /usr/local/share/fks/texresults-vyfuk.xsl
-XSL_STATS := /usr/local/share/fks/texstats.xsl
-XSL_CSV := /usr/local/share/fks/export2csv.xsl
+XSL_SIGNATURES := /usr/share/fks/texsignatures.xsl
+#XSL_RESULTS := /usr/share/fks/texresults-fykos.xsl
+#XSL_RESULTS := /usr/share/fks/texresults-vyfuk.xsl
+XSL_STATS := /usr/share/fks/texstats.xsl
+XSL_CSV := /usr/share/fks/export2csv.xsl
```

Comment:
Hm, this will have effect for all TeX `make` users.
Cc: @MeggyCal @Auratons and Astrid will need update their Makefile.conf for proper compilation.